### PR TITLE
Stats Refresh: Follower totals block

### LINKED ISSUE #9460: Create Follower totals block in Stats
Create a Follower totals block in the Refreshed stats

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #9460.

This PR adds a new Follower totals block.

image

To test:

1. Go to Stats
2. Tap on the Insights tab
3. Scroll down and notice there is a new Follower Totals block

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowerTotalsUseCase.kt [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.FollowerTotalsUseCase.FollowerType
+import org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.FollowerTotalsUseCase.FollowerType.EMAIL
+import org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.FollowerTotalsUseCase.FollowerType.SOCIAL
+import org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.FollowerTotalsUseCase.FollowerType.WP_COM
+import org.wordpress.android.ui.stats.refresh.utils.StatsSiteProvider
+import org.wordpress.android.ui.stats.refresh.utils.toFormattedString
+import javax.inject.Inject
+import javax.inject.Named
+
+class FollowerTotalsUseCase
```
Comment (planarvoid) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowerTotalsUseCase.kt]: do you think you can write a unit test for this use case?

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowerTotalsUseCase.kt [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    private val followersStore: FollowersStore,
+    private val publicizeStore: PublicizeStore,
+    private val statsSiteProvider: StatsSiteProvider
+) : StatelessUseCase<Map<FollowerType, Int>>(FOLLOWER_TOTALS, mainDispatcher) {
+    override fun buildLoadingItem(): List<BlockListItem> = listOf(Title(R.string.stats_view_follower_totals))
+
+    override suspend fun loadCachedData(): Map<FollowerType, Int>? {
+        val wpComFollowers = followersStore.getWpComFollowers(statsSiteProvider.siteModel, LimitMode.Top(0))
+        val emailFollowers = followersStore.getEmailFollowers(statsSiteProvider.siteModel, LimitMode.Top(0))
+        val publicizeServices = publicizeStore.getPublicizeData(statsSiteProvider.siteModel, LimitMode.All)
```
Comment (planarvoid) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowerTotalsUseCase.kt]: This is an interesting approach but I like it! Using more stores in one use-case

**Review by planarvoid — APPROVED** [review]
Looks great! 👍

**Review by iadeelzafar — APPROVED** [review]
Awesome work. LGTM.
P.S: Resolve the merge conflicts.

**Review by planarvoid — APPROVED** [review]
Looks great :shipit:

**Pull request merged**